README.md

```diff
@@ -0,0 +1,130 @@
+# Hotel Booking frontend-Project
+#### Link to Rails App [Hotel Booking Backend](https://github.com/angeluray/hotel-booking-backend.git)
+
+# Description  
+Hotel Booking Frontend is a Hotel Booking application where the user can register new accounts, login and reserve a set of different Hotels. It is built and connected by using two different repos, including Back-end(Rails) and Front-end(React/Redux).
+
+## Built With
+- HTML
+- JS
+- Tailwind
+- CSS
+- JSX
+- ReactJS
+- Redux
+
+## Screenshots
+
+![image_2023-01-19_19-00-48](https://user-images.githubusercontent.com/97189760/213582355-7102d3bb-4878-4fe2-8207-ddb6ac574844.png)
+![image_2023-01-19_19-01-17](https://user-images.githubusercontent.com/97189760/213582378-7a3e0f7f-f044-4f04-a30f-bba6f60cb7ec.png)
+![image_2023-01-19_19-01-39](https://user-images.githubusercontent.com/97189760/213582402-44439e5e-4efb-468e-ae6c-84c791fc7bcb.png)
+![image_2023-01-19_19-02-40](https://user-images.githubusercontent.com/97189760/213582424-5e6044ab-6a6c-413c-9366-2ad717f7cb4e.png)
+![image_2023-01-19_19-03-41](https://user-images.githubusercontent.com/97189760/213582468-966eea6e-e585-467a-aa87-99c861f6b4dd.png)
+![image_2023-01-19_19-04-03](https://user-images.githubusercontent.com/97189760/213582560-a72aaa97-70a9-40a1-a751-4165db79099f.png)
+![image_2023-01-19_19-06-00](https://user-images.githubusercontent.com/97189760/213582599-cd6585ad-80df-4ac3-8e15-f12a3abe3006.png)
+
+## Getting Started
+
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+ - IDE(code editor) like: **Vscode**, **Sublime**, etc.  
+ - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+
+### To launch the project locally:
+
+#### Step 1:
+- Clone this repo locally using git and the following git command
+
+  ```sh
+  git clone https://github.com/angeluray/hotel-booking-frontend.git
+  ```
+
+#### Step 2:
+
+ ```sh
+  npm start
+```
```

Comment:
- I invite you to add the step `npm install` before `npm start`. 🐶

Reply:
Thanks @vichuge, We have done it.

Reply:
**Added placeholder to avoid confusion to create hotels and add images**

![image](https://user-images.githubusercontent.com/65109017/213659464-31e7adff-6313-42b7-84c3-a8e694d57d19.png)